infra/notifications/src/app.js

```diff
@@ -329,7 +329,7 @@ app.post('/events', async (req, res) => {
   const { event = {}, related = {} } = req.body
   const { returnValues = {} } = event
   const eventName = event.event
-  const { listing, offer } = related
+  const { listing = {}, offer = {} } = related
```

Comment:
Is this valid? I didn't think defaults were legit for object unpacking.

Reply:
Yeah. Seems to work... ah the joy of JS ! ;)
```
> const { listing = {}, offer = {} } = {}
undefined
> listing
{}
> offer
{}

```